client/dashboard/utils/domain-types.ts

```diff
@@ -4,3 +4,11 @@ import type { DomainSummary } from '@automattic/api-core';
 export function isTransferrableToWpcom( domain: DomainSummary ) {
 	return domain.subtype.id === DomainSubtype.DOMAIN_CONNECTION;
 }
+
+export const domainTypes = {
```

Comment:
I think these already exist as `DomainTypes` in `packages/api-core` but they're lowercase, so something might need to be adjusted to use them correctly.